src/organization/organization.selectors.js

```diff
@@ -0,0 +1,48 @@
+import { createSelector } from 'reselect';
+
+const getOrganizationFromStore = state => state.organization;
+
+export const getOrganization = createSelector(
+  getOrganizationFromStore,
+  organization => organization.organization || {}
```

Comment:
I was having a disussion with colleagues from work this past week and a few were under the impression that when simply getting object parameters with dot notation doesn't necessarily _need_ `createSelector`. For example, this would be sufficient:

```
export const getOrganization = state => state.organization.organization;
```

Not necessarily sure if I agree however. Majority of all our selectors are simple getters and not involve manipulation of data in anyway so it seems if we went with this approach we wouldn't really be using `createSelector` for the majority of our state.

Just wanted to get your input on this mate. Do you usually use `createSelector` for all data or _only_ if you're retrieving data and performing manipulation to it?